data/moves.ts

```diff
@@ -502,8 +502,11 @@ export const Moves: {[moveid: string]: MoveData} = {
 		flags: {snatch: 1, distance: 1},
 		onHit(pokemon, source, move) {
 			this.add('-activate', source, 'move: Aromatherapy');
+			const side = pokemon.side;
+			let allies = side.pokemon;
+			if (side.allySide) allies = allies.concat(side.allySide.pokemon);
 			let success = false;
-			for (const ally of pokemon.side.pokemon) {
+			for (const ally of allies) {
```

Comment:
```suggestion
			for (const ally of pokemon.side.allies()) {
```

Reply:
yeah the issue with #Pokemon.allies is that it only gets active allies. Maybe we could make a new function that gets inactive ones as well?

Reply:
Sorry, I forgot that the `allyTeam` target doesn't actually target the team, it's just for show, and Aromatherapy and Heal Bell actually have to manually iterate the team. (I don't know whether it would be possible to fix that; you would need to activate the move in a separate event if you did.)